### PR TITLE
Add LO PDF info

### PR DESCRIPTION
**Pull request by rappoccio**
This adds the LO PDF info for pdf reweighting later. This can be used with LHAPDF prescriptions (caveat emptor, the user must be aware of what they are doing physics-wise, this is only well defined for LO samples).

### CODE REVIEW

File: PhysicsTools/NanoAOD/python/globals_cff.py
Comment (gpetruc) [inline PhysicsTools/NanoAOD/python/globals_cff.py]: can you fix the help strings?

Comment (gpetruc) [reply PhysicsTools/NanoAOD/python/globals_cff.py]: also can you check the difference between x and xpdf and understand which of the two is the one to keep? (i don't think we need both...)

**Review by gpetruc — CHANGES_REQUESTED** [review]

File: PhysicsTools/NanoAOD/python/globals_cff.py
Comment (rappoccio) [reply PhysicsTools/NanoAOD/python/globals_cff.py]: We need both, x is the momentum fraction, xpdf is "x * pdf(x)". However, it looks like they are often set to zero for our production.

**Review by gpetruc — APPROVED** [review]

**Comment by arizzi** [comment]
@gpetruc I assume the size increase is due to your PR that is included here
(i.e. master was not rerun) ... right?

**Comment by gpetruc** [comment]
yes

**Comment by arizzi** [comment]
for better tracking of features-PRs I'd like to rerun the CI on master and then again on this one before merging. 
I'll do it if/when you are done testing the 10XX stuff... so please do not merge this before

**Pull request merged**
